contracts/src/CrossChainApplications/examples/NativeTokenBridge/NativeTokenDestination.sol

```diff
@@ -84,10 +147,14 @@ contract NativeTokenDestination is TeleporterOwnerUpgradeable, INativeTokenDesti
 
         initialReserveImbalance = initialReserveImbalance_;
         currentReserveImbalance = initialReserveImbalance_;
+
+        require(decimalsShift <= 18, "NativeTokenDestination: invalid decimalsShift");
+        tokenMultiplier = 10 ** decimalsShift;
+        multiplyOnReceive = multiplyOnReceive_;
     }
 
     /**
-     * @dev See {INativeTokenDestination-transferToSource}.
+     * @inheritdoc INativeTokenDestination
```

Comment:
nit: I think we should keep this consistent throughout repo. Would recommend leaving and creating new issue, or changing it throughout repo.